### PR TITLE
Fix batch files

### PR DESCRIPTION
**Pull request by daxgames**
Replaced %CMDER_ROOT%\ with %CMDER_ROOT% because the variable already ends with \

### CODE REVIEW

File: Cmder.bat [hunk 1]
```diff
@@ -1,3 +1,3 @@
 @echo off
 SET CMDER_ROOT=%~dp0
-start %~dp0/vendor/conemu-maximus5/ConEmu.exe /Icon "%CMDER_ROOT%\icons\cmder.ico" /Title Cmder /LoadCfgFile "%CMDER_ROOT%\config\ConEmu.xml"
+start %~dp0/vendor/conemu-maximus5/ConEmu.exe /Icon "%CMDER_ROOT%icons\cmder.ico" /Title Cmder /LoadCfgFile "%CMDER_ROOT%config\ConEmu.xml"
```
Comment (JanSchulz) [inline Cmder.bat]: This looks like an oversight: the normal launcher does not use /LoadCfgFile anymore

**Comment by daxgames** [comment]
I am also working on enabling the use of the new  git bash.  Should be ready soon

**Comment by daxgames** [comment]
It will be a separate pull request though

**Comment by Stanzilla** [comment]
I don't even know if we want to keep the bat around, it's only used for Windows XP anyway, no?

**Comment by daxgames** [comment]
It is in the source so i fixed it.  Do with it whatever you feel is necessary.

**Comment by Stanzilla** [comment]
I think it looks weird without the `\`in the paths :/

**Comment by daxgames** [comment]
It might look wierd but it is correct.

It looks even wierder when you look at the %path% or a variable that has 
'\\' in it.

**Comment by Stanzilla** [comment]
Well actually it might be the better way to just remove the `\` from the variable, don't you think?

**Comment by daxgames** [comment]
We could do that to get rid of the wierdness.

That would also require changes to my other pull request as it behaves in a manner consistent with what init.bat does and expects %CMDER_ROOT% and $CMDER_ROOT both end with either a '\' or '/'.

**Comment by Stanzilla** [comment]
@cmderdev/trusted-contributors @cmderdev/owners any objections to changing it like that?

Also @daxgames can we leave https://github.com/daxgames/cmder/commit/e56e00669113b9bad3d57b19a3623ebd445191b2 out of this PR, I don
t really think we need your personal files in the .gitignore, I know it does not hurt either but yeah.

**Comment by daxgames** [comment]
I actually modified the .gitignore in my last commit so it is relevant to all and does not contain any personal files.  I only added the config/user-* files that are auto created by launching shells in this build.

**Comment by Stanzilla** [comment]
Nope, that's good.

**Comment by daxgames** [comment]
Squashed

**Comment by daxgames** [comment]
hang on something got messed up in the squash let me fix it.

**Comment by daxgames** [comment]
Squash fixed.  One commit 3 files changed.  Sorry about that somehow I combined two branches. in the previous squash

**Comment by Stanzilla** [comment]
Great, thank you!

**Pull request merged**